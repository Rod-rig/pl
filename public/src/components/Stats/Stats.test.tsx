import * as React from 'react';
import {Stats} from './';

describe('Stats', () => {
  it('test', () => {
    expect(1 + 1).toBe(2);
  });
});
