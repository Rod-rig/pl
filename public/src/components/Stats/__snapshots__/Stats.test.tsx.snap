// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stats should render empty stats comp correctly 1`] = `
<div
  className="MuiPaper-root-5 MuiPaper-elevation2-9 MuiPaper-rounded-6 Component-paper-124"
>
  <svg
    aria-hidden="true"
    className="MuiSvgIcon-root-125 MuiSvgIcon-colorPrimary-126 MuiSvgIcon-fontSizeLarge-133"
    focusable="false"
    role="presentation"
    viewBox="0 0 24 24"
  >
    <path
      d="M12 22c1.1 0 2-.9 2-2h-4c0 1.1.89 2 2 2zm6-6v-5c0-3.07-1.64-5.64-4.5-6.32V4c0-.83-.67-1.5-1.5-1.5s-1.5.67-1.5 1.5v.68C7.63 5.36 6 7.92 6 11v5l-2 2v1h16v-1l-2-2z"
    />
  </svg>
  <h4
    className="MuiTypography-root-32 MuiTypography-h4-47 MuiTypography-gutterBottom-59"
  >
    No statistics
  </h4>
  <p
    className="MuiTypography-root-32 MuiTypography-body1-41 MuiTypography-gutterBottom-59"
  >
    Looks like you have no predictions yet
  </p>
  <div
    className="Component-links-123"
  >
    <a
      className="MuiTypography-root-32 MuiTypography-colorSecondary-63 MuiTypography-inline-67 MuiLink-root-134 MuiLink-underlineHover-136"
      href="/"
      onClick={[Function]}
    >
      Go to main page
    </a>
    <a
      className="MuiTypography-root-32 MuiTypography-colorSecondary-63 MuiTypography-inline-67 MuiLink-root-134 MuiLink-underlineHover-136"
      href="/predictions"
      onClick={[Function]}
    >
      Make predictions
    </a>
  </div>
</div>
`;

exports[`Stats should render stats with not loaded predictions correctly 1`] = `
<div
  className="MuiCircularProgress-root-113 Component-root-112 MuiCircularProgress-colorPrimary-116 MuiCircularProgress-indeterminate-115"
  role="progressbar"
  style={
    Object {
      "height": 40,
      "width": 40,
    }
  }
>
  <svg
    className="MuiCircularProgress-svg-118"
    viewBox="22 22 44 44"
  >
    <circle
      className="MuiCircularProgress-circle-119 MuiCircularProgress-circleIndeterminate-121"
      cx={44}
      cy={44}
      fill="none"
      r={20.2}
      strokeWidth={3.6}
      style={Object {}}
    />
  </svg>
</div>
`;

exports[`Stats should render stats with predictions correctly 1`] = `
Array [
  <div
    className="Component-wrapper-4"
  >
    <div
      className="MuiPaper-root-5 MuiPaper-elevation2-9 MuiPaper-rounded-6 Component-paper-2"
    >
      <p
        className="MuiTypography-root-32 MuiTypography-body1-41 MuiTypography-gutterBottom-59 Component-text-3"
      >
        Predictions success %
      </p>
      <div
        className="Component-number-1"
      >
        33.33
      </div>
    </div>
    <div
      className="MuiPaper-root-5 MuiPaper-elevation2-9 MuiPaper-rounded-6 Component-paper-2"
    >
      <p
        className="MuiTypography-root-32 MuiTypography-body1-41 MuiTypography-gutterBottom-59 Component-text-3"
      >
        Total predictions
      </p>
      <div
        className="Component-number-1"
      >
        4
      </div>
    </div>
    <div
      className="MuiPaper-root-5 MuiPaper-elevation2-9 MuiPaper-rounded-6 Component-paper-2"
    >
      <p
        className="MuiTypography-root-32 MuiTypography-body1-41 MuiTypography-gutterBottom-59 Component-text-3"
      >
        Predicted correctly
      </p>
      <div
        className="Component-number-1"
      >
        1
      </div>
    </div>
    <div
      className="MuiPaper-root-5 MuiPaper-elevation2-9 MuiPaper-rounded-6 Component-paper-2"
    >
      <p
        className="MuiTypography-root-32 MuiTypography-body1-41 MuiTypography-gutterBottom-59 Component-text-3"
      >
        Predictions pending
      </p>
      <div
        className="Component-number-1"
      >
        1
      </div>
    </div>
  </div>,
  <div
    className="MuiButtonBase-root-91 MuiListItem-root-79 MuiListItem-default-82 MuiListItem-gutters-87 MuiListItem-divider-86 MuiListItem-button-88 Component-matchItem-75"
    onBlur={[Function]}
    onContextMenu={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex="0"
  >
    <div
      className="MuiListItemText-root-94 Component-text-78 Component-home-73"
    >
      <div>
        Newcastle United
      </div>
      <div
        className="MuiListItemIcon-root-101 Component-homeIcon-74"
      >
        <div
          className="MuiAvatar-root-102 Component-logo-100"
        >
          <img
            className="MuiAvatar-img-104"
            src="https://upload.wikimedia.org/wikipedia/en/5/56/Newcastle_United_Logo.svg"
          />
        </div>
      </div>
    </div>
    <div
      className="Component-score-77 Component-red-76"
    >
      <div>
        2
      </div>
      <div
        className="Component-dash-69"
      >
        :
      </div>
      <div>
        2
      </div>
    </div>
    <div
      className="MuiListItemText-root-94 Component-text-78 Component-guest-72"
    >
      <div
        className="MuiListItemIcon-root-101 Component-awayIcon-68"
      >
        <div
          className="MuiAvatar-root-102 Component-logo-100"
        >
          <img
            className="MuiAvatar-img-104"
            src="https://upload.wikimedia.org/wikipedia/en/b/b4/Tottenham_Hotspur.svg"
          />
        </div>
      </div>
      <div>
        Tottenham Hotspur
      </div>
    </div>
    <span
      className="MuiTouchRipple-root-105"
    />
  </div>,
  <div
    className="MuiButtonBase-root-91 MuiListItem-root-79 MuiListItem-default-82 MuiListItem-gutters-87 MuiListItem-divider-86 MuiListItem-button-88 Component-matchItem-75"
    onBlur={[Function]}
    onContextMenu={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex="0"
  >
    <div
      className="MuiListItemText-root-94 Component-text-78 Component-home-73"
    >
      <div>
        Liverpool FC
      </div>
      <div
        className="MuiListItemIcon-root-101 Component-homeIcon-74"
      >
        <div
          className="MuiAvatar-root-102 Component-logo-100"
        >
          <img
            className="MuiAvatar-img-104"
            src="https://upload.wikimedia.org/wikipedia/en/0/0c/Liverpool_FC.svg"
          />
        </div>
      </div>
    </div>
    <div
      className="Component-score-77 Component-green-71"
    >
      <div>
        1
      </div>
      <div
        className="Component-dash-69"
      >
        :
      </div>
      <div>
        2
      </div>
    </div>
    <div
      className="MuiListItemText-root-94 Component-text-78 Component-guest-72"
    >
      <div
        className="MuiListItemIcon-root-101 Component-awayIcon-68"
      >
        <div
          className="MuiAvatar-root-102 Component-logo-100"
        >
          <img
            className="MuiAvatar-img-104"
            src="https://upload.wikimedia.org/wikipedia/en/c/c2/West_Ham_United_FC_logo.svg"
          />
        </div>
      </div>
      <div>
        West Ham United
      </div>
    </div>
    <span
      className="MuiTouchRipple-root-105"
    />
  </div>,
  <div
    className="MuiButtonBase-root-91 MuiListItem-root-79 MuiListItem-default-82 MuiListItem-gutters-87 MuiListItem-divider-86 MuiListItem-button-88 Component-matchItem-75"
    onBlur={[Function]}
    onContextMenu={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex="0"
  >
    <div
      className="MuiListItemText-root-94 Component-text-78 Component-home-73"
    >
      <div>
        Wolverhampton Wanderers
      </div>
      <div
        className="MuiListItemIcon-root-101 Component-homeIcon-74"
      >
        <div
          className="MuiAvatar-root-102 Component-logo-100"
        >
          <img
            className="MuiAvatar-img-104"
            src="https://upload.wikimedia.org/wikipedia/en/f/fc/Wolverhampton_Wanderers.svg"
          />
        </div>
      </div>
    </div>
    <div
      className="Component-score-77 Component-default-70"
    >
      <div>
        6
      </div>
      <div
        className="Component-dash-69"
      >
        :
      </div>
      <div>
        1
      </div>
    </div>
    <div
      className="MuiListItemText-root-94 Component-text-78 Component-guest-72"
    >
      <div
        className="MuiListItemIcon-root-101 Component-awayIcon-68"
      >
        <div
          className="MuiAvatar-root-102 Component-logo-100"
        >
          <img
            className="MuiAvatar-img-104"
            src="https://upload.wikimedia.org/wikipedia/en/e/eb/Manchester_City_FC_badge.svg"
          />
        </div>
      </div>
      <div>
        Manchester City
      </div>
    </div>
    <span
      className="MuiTouchRipple-root-105"
    />
  </div>,
  <div
    className="MuiButtonBase-root-91 MuiListItem-root-79 MuiListItem-default-82 MuiListItem-gutters-87 MuiListItem-divider-86 MuiListItem-button-88 Component-matchItem-75"
    onBlur={[Function]}
    onContextMenu={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex="0"
  >
    <div
      className="MuiListItemText-root-94 Component-text-78 Component-home-73"
    >
      <div>
        Cardiff City
      </div>
      <div
        className="MuiListItemIcon-root-101 Component-homeIcon-74"
      >
        <div
          className="MuiAvatar-root-102 Component-logo-100"
        >
          <img
            className="MuiAvatar-img-104"
            src="https://upload.wikimedia.org/wikipedia/en/3/3c/Cardiff_City_crest.svg"
          />
        </div>
      </div>
    </div>
    <div
      className="Component-score-77 Component-red-76"
    >
      <div>
        1
      </div>
      <div
        className="Component-dash-69"
      >
        :
      </div>
      <div>
        1
      </div>
    </div>
    <div
      className="MuiListItemText-root-94 Component-text-78 Component-guest-72"
    >
      <div
        className="MuiListItemIcon-root-101 Component-awayIcon-68"
      >
        <div
          className="MuiAvatar-root-102 Component-logo-100"
        >
          <img
            className="MuiAvatar-img-104"
            src="https://upload.wikimedia.org/wikipedia/en/e/eb/Fulham_FC_%28shield%29.svg"
          />
        </div>
      </div>
      <div>
        Fulham FC
      </div>
    </div>
    <span
      className="MuiTouchRipple-root-105"
    />
  </div>,
]
`;
